source/include/FreeRTOS_IP.h

```diff
@@ -142,12 +142,6 @@ extern uint32_t ulApplicationGetNextSequenceNumber( uint32_t ulSourceAddress,
  * handled.  The value is chosen simply to be easy to spot when debugging. */
 #define ipUNHANDLED_PROTOCOL    0x4321U
 
-/** @brief The maximum time the IP task is allowed to remain in the Blocked state if no
- * events are posted to the network event queue. */
-#ifndef ipconfigMAX_IP_TASK_SLEEP_TIME
-    #define ipconfigMAX_IP_TASK_SLEEP_TIME    ( pdMS_TO_TICKS( 10000UL ) )
-#endif
-
```

Comment:
Is this set to a default elsewhere in the repo?
Or is this now just an error not to provide it in `FreeRTOSIPConfig.h`?

If it is an error NOT to include this can this check be re-added to `FreeRTOSIP.h`?

Reply:
FreeRTOSIPConfigDefaults.h